src/services/worker/tasks/fetch.task.ts

```diff
@@ -11,11 +11,15 @@ import {
 
 @Injectable()
 export class FetchTask {
+  private _isProcessLocked: boolean;
+
   constructor(
     private _systemService: SystemService,
     private _workerService: WorkerService,
     private readonly _metricsService: MetricsService,
-  ) {}
+  ) {
+    this._isProcessLocked = false;
+  }
```

Comment:
Are you sure that FetchTask uses as singleton (or only in one place)?
Otherwise, this variable should be static